modules/ROOT/pages/tls-configuration.adoc

```diff
@@ -93,12 +93,18 @@ Follow these steps to generate a keystore and export a self-signed certificate.
 For example, run the following command:
 +
 ----
-keytool -genkey -alias <key-alias> -keystore <keystore-name>.jks
+keytool -genkey -keyalg RSA -alias <key-alias> -keystore <keystore-name>.jks
 ----
 +
-Replace `<keystore-name>` with the name you want for your keystore. +
-Replace `<key-alias>` with a unique alias of your choice. +
-Keytool generates certificates using the DSA algorithm by default, you can instead specify it to use the RSA algorithm by adding `-keyalg RSA` to the previous command.
+Replace `<key-alias>` with a unique alias of your choice.
++
+Replace `<keystore-name>` with the name you want for your keystore.
++
+You can change the encryption algorithm to ECDSA by specifying `-keyalg EC` instead of `-keyalg RSA`.
++
+[IMPORTANT]
+If you do not specify the algorithm, the tool uses DSA by default which is not compatible with TLS 1.2 and causes the SSL handshake to fail when the Mule app receives an HTTPS request.
+
```

Comment:
So here is what I think about this:

We have sample syntax and instructions to replace the variables within it to specific values.
But I don't see the algorithm syntax in the sample, even though we go on to describe it?
Can we add the encryption algorithm to the sample?

Also..."you can change" is not very specific: should this be parallel with the previous two statements and begin with "Replace"? Under what circumstances would you change to EC instead of RSA? This phrasing makes it sound like RSA is the default, but we go on to say that DSA is the default? How does this statement relate to the notice?

Do we really mean to tell the audience that it must specify an encryption algorithm, lest the tool default to DSA and cause failure, and that the algorithm must be one of EC or RSA (and how to choose which)?

Reply:
copyedit only
```suggestion
Change the encryption algorithm to ECDSA by specifying `-keyalg EC` instead of `-keyalg RSA`.
+
[IMPORTANT]
If you do not specify the encryption algorithm, the tool uses DSA by default, which is not compatible with TLS 1.2 and causes the SSL handshake to fail when the Mule app receives an HTTPS request.

```